frontend/README.md

```diff
@@ -12,41 +11,35 @@ You will need the following things properly installed on your computer.
 * [Ember CLI](https://ember-cli.com/)
 * [Google Chrome](https://google.com/chrome/)
 
-## Installation
-
-* `git clone <repository-url>` this repository
-* *`cd frontend`
-* *`npm install`
-  
-\* if you're using docker, these steps will not be necessary
-
-## Running / Development
-
-#### Without docker
+### Without docker
 
 * `ember serve`
 * Visit your app at [http://localhost:4200](http://localhost:4200).
 * Visit your tests at [http://localhost:4200/tests](http://localhost:4200/tests).
 
-#### With docker
+### With docker
 
 `docker-compose up -d`
 
-### Code Generators
+## Code Generators
 
 Make use of the many generators for code, try `ember help generate` for more details
 
-### Running Tests
+### Running tests
+To run the frontend tests you will have to run
 
-#### Without docker
+```
+$ ember test
+```
 
-* `ember test`
-* `ember test --server`
+or 
 
-#### With docker
+```
+$ ember test --server
```

Comment:
why do we have to add --server when using docker setup?
how to connect to the docker container's console?
wouldn't it be ok to only supporting the docker setup from now on?